src/components/popover/popover.tsx

```diff
@@ -42,4 +66,12 @@ export class Popover {
             </limel-portal>
         );
     }
+
+    private globalClickListener({ target }) {
+        const clickedInside = Boolean(target.closest(`#${this.portalId}`));
```

Comment:
Nice didn't know you could do stuff like this with `closest` :slightly_smiling_face: